.github/workflows/cln-plugin.yaml

```diff
@@ -38,6 +38,7 @@ jobs:
           sudo apt-get update && sudo apt-get install -y gettext
           git clone https://github.com/ElementsProject/lightning.git && cd lightning && git checkout v${{ env.cln_version }}
           pip install --user poetry && poetry install
+          cargo update -p time
```

Comment:
but now the versions are no longer locked!

this usually is more problematic. we don't always get the case where a change in rust version breaks something (happened today though :/), but the case where a change in the package version breaks something happens more often.

I would argue even that the first case should be addressed swiftly anyway because a crate doesn't want to not support a recent rust version ofc.

Reply:
I think this should be ok given it is only used to create the cache, so as long as we do not update the bitcoind or cln versions, the `time` version will be "locked".

We can revert once the issue is fixed upstream and remove this workaround

Reply:
> so as long as we do not update the bitcoind or cln versions, the time version will be "locked".

No, i think this will update `time` when a new version of `time` shows up.

That said, I confused what this `cargo update` was for. I thought it was for a toes crate (thus we can just run it locally to get a new lock file and push it on github so it's updated and locked), but since it's for `cln`, I think we should fix that in `cln` repo.

Reply:
This is run only when there is a cache miss, so a new cache is created. Caches are named after the os version, bitcoind version and con version, which are all locked, so the cache creation should happen only once (or once every time GitHub decides to delete the cache files).

I opened an issue upstream so this can be addressed